bdk-ffi/src/bdk.udl

```diff
@@ -1,4 +1,10 @@
-namespace bdk {};
+namespace bdk {
+  /// Build a BIP 157/158 light client to fetch transactions for a [`Wallet`].
+  [Throws=LightClientBuilderError]
+  NodePair build_light_client([ByRef] Wallet wallet, sequence<Peer> peers, u8 connections, u32? recovery_height, string data_dir);
+  /// Run a [`LightNode`] continuously on a detached OS thread.
+  void run_node(LightNode node);
```

Comment:
I think `run_node` is probably the better option for most, as it spawns a new OS thread and runs the node there, then immediately returns. I would expect this would be the workflow for the majority of use cases that just want to run it and forget about it, and it handles the sometimes complicated semantics of spawning threads in languages like python. That being said I think we should also keep `LightNode.run` in the public API because languages like swift that have the concept of a `Task` with different "priorities" might be able to force the node to stay alive even when the app is idle for instance.

Reply:
Okay. As we uncovered offline, we cannot expose `LightNode.run`, and will have to stick with `run_node`

Reply:
Would it make sense to move this to `Client::run_node` or is that non-intuitive?